src/hooks/queries/streamer.ts

```diff
@@ -0,0 +1,18 @@
+import { useMutation } from '@tanstack/react-query';
+
+import { postDailyMessage } from '@/apis/streamer';
+
+interface IPostDailyMessageProps {
+  setMessage: (value: string) => void;
+}
+
+export const usePostDailyMessage = ({ setMessage }: IPostDailyMessageProps) => {
+  return useMutation({
+    mutationFn: (data: IPostDailyMessageReq) => postDailyMessage(data),
```

Comment:
```suggestion
    mutationFn: postDailyMessage,
```

이렇게 단순화 할 수도 있을 것 같습니다!